code/game/objects/items.dm

```diff
@@ -530,3 +539,13 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 
 /obj/item/proc/is_equivalent(obj/item/I)
 	return I == src
+
+/obj/item/Crossed(atom/movable/AM)
+	if(prob(trip_chance) && ishuman(AM))

```

Comment:
This is a question I don't have an answer to. if the `if(prob(trip_chance))` returns `FALSE`, does the `ishuman(AM)` check still happen or not?